common/src/main/java/com/wynntils/features/user/overlays/CustomBarsOverlayFeature.java

```diff
@@ -183,6 +189,172 @@ public void render(PoseStack poseStack, float partialTicks, Window window) {
         }
     }
 
+    public static class AwakenedProgressBarOverlay extends Overlay {
+        @Config(key = "feature.wynntils.customBarsOverlay.overlay.awakenedProgressBar.textShadow")
+        public FontRenderer.TextShadow textShadow = FontRenderer.TextShadow.OUTLINE;
+
+        @Config(key = "feature.wynntils.customBarsOverlay.overlay.awakenedProgressBar.flip")
+        public boolean flip = false;
+
+        @Config(key = "feature.wynntils.customBarsOverlay.overlay.awakenedProgressBar.textColor")
+        public CustomColor textColor = CommonColors.WHITE;
+
+        public AwakenedProgressBarOverlay() {
+            super(
+                    new OverlayPosition(
+                            -70,
+                            -150,
+                            VerticalAlignment.Bottom,
+                            HorizontalAlignment.Center,
+                            OverlayPosition.AnchorSection.BottomMiddle),
+                    new GuiScaledOverlaySize(81, 21));
+        }
+
+        @Override
+        public void render(PoseStack poseStack, float partialTicks, Window window) {
+            if (!WynnUtils.onWorld()) return;
+
+            WynnBossBarUtils.AwakenedBar awakenedBar = WynnBossBarUtils.getAwakenedBar();
+            if (awakenedBar.equals(WynnBossBarUtils.NO_AWAKENED_PROGRESS)) return;
+
+            Texture universalBarTexture = Texture.UNIVERSAL_BAR;
+
+            final float renderedHeight = universalBarTexture.height() / 2f * (this.getWidth() / 81);
+
+            float renderY =
+                    switch (this.getRenderVerticalAlignment()) {
+                        case Top -> this.getRenderY();
+                        case Middle -> this.getRenderY() + (this.getHeight() - renderedHeight) / 2;
+                        case Bottom -> this.getRenderY() + this.getHeight() - renderedHeight;
+                    };
+
+            String text = "Awakened: " + awakenedBar.current() + "/200";
```

Comment:
I would kinda like the max value to be parsed from the boss bar like everything else does.

Reply:
Also this text is pretty large for the overlay, although this is not a breaking issue.